partiql-planner/build.gradle.kts

```diff
@@ -0,0 +1,72 @@
+import org.jetbrains.dokka.utilities.relativeTo
```

Comment:
Let's remove FUNCTIONS.adoc and PlannerCheckList.adoc for now.

Reply:
Removed.